packages/compiler/core/cli.ts

```diff
@@ -1,4 +1,4 @@
 /**
- * For backward compatibility where global compiler would look at ./node_modules/@cadl-lang/compiler/dist/core/cli/cli.js
+ * For backward compatibility where global compiler would look at ./node_modules/@typespec/compiler/dist/core/cli/cli.js
```

Comment:
I think we are already going to break the ability to resolve to here since we will use @cadl-lang package name when resolving, so maybe we can delete this now.

Reply:
No arbitrary modification this time. I will record this as a potential manual fix.

Reply:
(This is the one file that can be deleted now.)